src/commands/start/index.ts

```diff
@@ -71,6 +79,12 @@ export default class Start extends Command {
     }
 
     this.log(WELCOME_MESSAGE)
-    bootstrap()
+    const startupService = new StartupService()
+    if (args.firstArg !== 'server') {
```

Comment:
npx codeclimbers start with start the server through a daemon and enable it as a startup service